index.html

```diff
@@ -0,0 +1,150 @@
+        <span class="program-span">Main Program</span>
+        <hr class="line">
+
+        <div class="program-img">
+          <div class="lecture">
+            <img src="images/program_icon_01.png" alt="">
+            <h3 class="text">Lecture</h3>
+            <p class="p-tag">Listen to the speakers from various countries about the messages of sharing</p>
+          </div>
+          <div class="lecture">
+            <img src="images/program_icon_02.png" alt="">
+            <h3 class="text">Exhibition</h3>
+            <p class="p-tag"> Appreciate creations applying CC of artists, organised from Art Center</p>
+          </div>
+          <div class="lecture">
+            <img src="images/program_icon_03.png" alt="">
+            <h3 class="text">Forum</h3>
+            <p class="p-tag">Have the time to share your thoughts and opinions with experts for each topic</p>
+          </div>
+          <div class="lecture">
+            <img src="images/program_icon_04.png" alt="">
+            <h3 class="text">Workshop</h3>
+            <p class="p-tag">Try creating your own work using open source license rather than just watching</p>
+          </div>
+          <div class="lecture">
+            <img src="images/program_icon_05.png" alt="">
+            <h3 class="text">CC ignite</h3>
+            <p class="p-tag">Get opportunites to network with affiliates around, after the summit</p>
+          </div>
+        </div>
+        <div class="first-btn-con">
+          <button class="first-btn"><a href="#"> Join us @ Recre&te</a> </button>
+          <li class="program-detail"><a href="#">See the whole program</a></li>
+        </div>
+      </div>
+    </section>
+
+    <section id="section2">
+      <div class="main-container">
+        <div class="presenters">
+          <h2 class="main-program2">Featured Speakers</h2>
+          <hr class="line">
+          <div id="speakers-grid">
+          </div>
+          <div class="view-btn"><button id="viewmore">see more speakers</button></div>
+
+        </div>
+      </div>
+    </section>
+
+    <section class="connect-sections">
+      <div class="main-container">
+        <h2 class="main-program2">Partners</h2>
+        <hr class="line">
+        <div class="connect-img">
+          <img class="the-img" src="images/mozilla.png" alt="">
+          <img class="the-img" src="images/Google_grey.png" alt="">
+          <img class="the-img" src="images/naver.png" alt="">
+          <img class="the-img" src="images/kakao-2.png" alt="">
+        </div>
+      </div>
+    </section>
+
+    <section class="footer" id="footer">
+      <div class="main-container">
+        <div id="footer-con">
+          <div class="logo">
+            <a href="index.html"><img class="my-logo" src="images/my-logo.png" alt=""></a>
+          </div>
+          <div>
+            <p class="down" id="changes">2022 Summit. Some Rights Reserved.</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] It is required to produce a close replica of the guiding template. This includes color, font, background-color, margin, padding among others. The height of both the partner and footer sections on the guiding template are the same, but your design looks totally different from that. Kindly look into it and fix it.

[your-home-footer-page.webm](https://user-images.githubusercontent.com/26597432/203830317-26c7e003-ab50-4812-bd10-e621e593de96.webm)